### PR TITLE
A couple of issues solved

### LINKED ISSUE #4: Replace Contact Us Form
Replace the contact-us form with a static div displaying `email ID` etc.

### LINKED ISSUE #5: Sponsors div
Replace `TONY&GUY` sponsor with a stock image, maybe make a custom image saying `This place is reserved for sponsors`

### LINKED ISSUE #6: Login Popup
The border and the color of the Login button is both same which is not a good UI, replace the `Login` text with maybe white

image

### LINKED ISSUE #8: Login  modal doesnt Disappear
When Register is clicked after login, login modal still sticks around in the background

### PR DESCRIPTION
**Pull request by DeveloperAbhirupa**
Please review.

Fixes #8 , fixes #6 , fixes #5 , fixes #4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -485,47 +490,21 @@ <h4>Questions</h4>
               <div class="container">
                 <div class="section-title center">
                   <h2>Sponsored by</h2>
-                  <hr>
-                  <a href="https://www.facebook.com/toniandguy.vellore/" target=_blank><img src="img/tony_logo.png" height="400" id="tony"></a>
+                  <hr><br>
+                  <a href="#" target=_blank><img src="img/img01.jpg" id="tony"></a>
```
Comment (kira0204) [inline index.html]: Let's replace the id with something more understandable?

File: index.html [hunk 2]
```diff
@@ -485,47 +490,21 @@ <h4>Questions</h4>
...
                   <div class="rules-text" >
-                  </div>
+                  </div><br><br>
```
Comment (kira0204) [inline index.html]: Would prefer using margin/padding instead of repeated `<br>`, Might break the Mobile UI

File: index.html [hunk 3]
```diff
@@ -485,47 +490,21 @@ <h4>Questions</h4>
...
-                      </div>
-                      <div id="success"></div>
-                      <button type="submit" class="dark_btn btn btn-default" id="contact_btn">Send Message</button>
-                    </form>
-                  </div>
-                </div>
+              <div class="container-fluid" align="middle"><br>
+                Contact<hr><br>
+                Name: POPCORN<br><br>
+                E-mail Id: abc@xyz.com<br><br>
```
Comment (kira0204) [inline index.html]: Same goes here, use `<p>` tags etc instead of `<br>`

File: js/main.js [hunk 4]
```diff
@@ -32,9 +32,21 @@ function main() {
...
     var popup = document.getElementById("myPopup");
+    console.log("CLICKED MYbTN");
+    if ( $('.popup').css('display') == 'none' ){
+        tag=1;
+    }
+    if(tag==1)
+    {
+        tag=0;
+        popup.classList.toggle("show");
+    }
```
Comment (kira0204) [inline js/main.js]: This looks decent, we will clean this further later on.

File: index.html [hunk 5]
```diff
@@ -31,11 +31,16 @@
...
     var span = document.getElementsByClassName("close")[0];
-
+    var tag=0;
     // When the user clicks on the button, open the modal
     btn.onclick = function() {
       modal.style.display = "block";
+      $('.popup').hide();
+      tag=1;
+      console.log("CLICKED");
+
```
Comment (kira0204) [inline index.html]: Let's move this JS code to it's dedicated file in the next PR?

**Comment by kira0204** [comment]
Also can you edit your original comment and mention which issues this PR fixes, will help to close the issues automatically, I hope you remember the keywords which were like `Fixes #<issue-no>`

**Comment by kira0204** [comment]
Any update on this?

**Comment by DeveloperAbhirupa** [comment]
I was on a tour for a week. I reached home today, I will get this done asap.

**Comment by kira0204** [comment]
Cool, thanks.
Also @goelashwin36 wanted to work on these issues as well, it would make sense to refer those issues which you are fixing by this PR so that we know you are solving them :)
Edit your original comment to include those fixes

**Comment by kira0204** [comment]
Also, one PR per issue is a good way to tackle issues. even if they are small, just so that they don't overlap, branching is an amazing option

**Comment by DeveloperAbhirupa** [comment]
I have given 1 PR for 4 issues, and have accordingly edited my initial comment. Please verify the same.

**Pull request merged**